### PR TITLE
Pod Security Admission.

### PR DESCRIPTION
**Pull request by jvanz**
## Description
Adds a documentation showing how user can use Kubewarden together with the Pod Security Admission from Kubernetes.

Fix https://github.com/kubewarden/kubewarden-controller/issues/260

### CODE REVIEW

**Comment by flavio** [comment]
I've done some changes to the article, can you please review it?

File: docs/tasksDir/pod-security-with-kubewarden.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+          max: 2000
+        - min: 4000
+          max: 5000
+    run_as_group:
+      rule: "RunAsAny"
+    supplemental_groups:
+      rule: "RunAsAny"
+EOF
+```
+
```
Comment (raulcabello) [inline docs/tasksDir/pod-security-with-kubewarden.md:234]: maybe we should let the user know he has to wait for the policy to be active before moving to the next step

**Review by raulcabello — COMMENTED** [review]
I tried the examples in my local cluster and I left some comments about things we can improve. The rest LGTM

File: docs/tasksDir/pod-security-with-kubewarden.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+title: ""
+---
+
+# Using Pod Security Admission with Kubewarden
+
+Starting from Kubernetes 1.25 release, the Pod Security Policies have been removed
+and have been replaced by [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/),
+also known as *"PSA"*.
+
+Pod Security Admission has been created to simplify the process of securing Pod
```
Comment (viccuad) [inline docs/tasksDir/pod-security-with-kubewarden.md:12]:
```suggestion
Pod Security Admission has been created to simplify the process of securing Pods
```

**Review by viccuad — APPROVED** [review]
LGTM modulo Raul's comments.

I mused about using a `baseline` as example (it may be the most common), but I think showcasing Kubewarden's mutation and so is better.

File: docs/tasksDir/pod-security-with-kubewarden.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+kind: ClusterAdmissionPolicy
+metadata:
+  name: user-group-psp
+spec:
+  policyServer: default
+  module: registry://ghcr.io/kubewarden/policies/user-group-psp:latest
+  rules:
+  - apiGroups: [""]
+    apiVersions: ["v1"]
+    resources: ["pods"]
```
[the remark this answers is not in the document]
Comment (flavio) [reply docs/tasksDir/pod-security-with-kubewarden.md:214]: Good point, thanks!

File: docs/tasksDir/pod-security-with-kubewarden.md (hunk 1 above)
Comment (flavio) [reply docs/tasksDir/pod-security-with-kubewarden.md:234]: done with d54a12124be4c50167b1ba742a663484b7123267

**Review by viccuad — APPROVED** [review]

**Pull request merged**